File: app/src/main/java/com/hussain/podcastapp/model/ApiResponse.java
package com.hussain.podcastapp.model;

import java.util.List;

public class ApiResponse {

    private Feed feed;

    public ApiResponse() {
    }

    public Feed getFeed() {
        return feed;
    }

    public void setFeed(Feed feed) {
        this.feed = feed;
    }

    public class Feed {
        private List<Entry> entry;

        public List getEntry() {
            return entry;
        }

        public void setEntry(List entry) {
            this.entry = entry;
        }

    }
}

File: app/src/main/java/com/hussain/podcastapp/model/ApiResponse.kt
package com.hussain.podcastapp.model

class ApiResponse {

    var feed: Feed? = null

    inner class Feed {
        private var entry: List<Entry>? = null

        fun getEntry(): List<Entry>? {
            return entry
        }

        fun setEntry(entry: List<Entry>) {
            this.entry = entry
        }

    }
}
